src/main/scala/org/constellation/consensus/CrossTalkConsensus.scala

```diff
@@ -20,7 +21,7 @@ import scala.concurrent.duration._
 
 class CrossTalkConsensus(remoteSenderSupervisor: ActorRef, config: Config)(implicit dao: DAO)
     extends Actor
-    with ActorLogging {
+    with StrictLogging {
```

Comment:
Why such change? Is there a difference in output?

Reply:
Log levels and format can be maintained from one file, also syntax will remain same at project level